torchbenchmark/models/doctr/__init__.py

```diff
@@ -0,0 +1,36 @@
+from doctr.models import ocr_predictor
+import numpy as np
+import torch
+
+# TorchBench imports
+from torchbenchmark.util.model import BenchmarkModel
+from torchbenchmark.tasks import COMPUTER_VISION
+
+from typing import Tuple
+
+class Model(BenchmarkModel):
+    task = COMPUTER_VISION.DETECTION
+    DEFAULT_EVAL_BSIZE = 2
+
+    def __init__(self, test, device, jit=False, batch_size=None, extra_args=[]):
+        super().__init__(test=test, device=device, jit=jit, batch_size=batch_size, extra_args=extra_args)
+
+        self.model = ocr_predictor(det_arch='db_resnet50', reco_arch='crnn_vgg16_bn', pretrained=True).to(self.device)
+        # fake document file
+        self.example_inputs = []
+        for _i in range(self.batch_size):
+            input_page = (255 * np.random.rand(600, 800, 3)).astype(np.uint8)
+            self.example_inputs.append(input_page)
+        if self.test == "eval":
+            self.model.eval()
+
+    def train(self):
+        raise NotImplementedError("Train is not implemented for this model.")
+
+    def get_module(self):
+        return self.model, self.example_inputs
+
+    def eval(self) -> Tuple[torch.Tensor]:
+        with torch.inference_mode():
+            out = self.model(self.example_inputs)
+        return out
```

Comment:
Is there a way to output tensor so that we can check correctness?

Reply:
in the forward pass add return_model_output=True and it will output the model output instead of the postprocessed output. 

change it to 
out = self.model(self.example_inputs, return_model_output=True)

Reply:
Use the argument to output tensor.